doc/api/net.md

```diff
@@ -205,34 +308,39 @@ persist*, they are removed when the last reference to them is closed. Do not
 forget JavaScript string escaping requires paths to be specified with
 double-backslashes, such as:
 
-    net.createServer().listen(
-        path.join('\\\\?\\pipe', process.cwd(), 'myctl'))
+```js
+net.createServer().listen(
+  path.join('\\\\?\\pipe', process.cwd(), 'myctl'))
+```
 
-The parameter `backlog` behaves the same as in
-[`server.listen(port[, hostname][, backlog][, callback])`][`server.listen(port, host, backlog, callback)`].
+The `backlog` argument is the maximum size of the queue of pending connections.
+The maximum possible value will be determined by the operating system through
+`sysctl` settings such as `tcp_max_syn_backlog` and `somaxconn` on Linux.
```

Comment:
I'm not sure this information is useful here. Maybe just `The maximum possible value is determined by the operating system` ?